app/src/main/java/com/example/warriorsocial/ui/organizations/OrganizationsFragment.java

```diff
@@ -92,10 +91,12 @@ protected void onBindViewHolder(OrganizationsViewHolder viewHolder, int position
                 System.out.println("inside onBindViewHolder in OrganizationsFragment");
                 final DatabaseReference StudentOrganizationRef = getRef(position);
                 // For Testing
+                /*
                 System.out.println("StudentOrganizationRef KEY: " + StudentOrganizationRef.getKey());
                 System.out.println("StudentOrganizationRef PARENT: " + StudentOrganizationRef.getParent());
                 System.out.println("StudentOrganizationRef REF: " + StudentOrganizationRef.getRef());
                 System.out.println("StudentOrganizationRef ROOT: " + StudentOrganizationRef.getRoot());
+                */
```

Comment:
Test Code was cleaned up but there is still many "print out statements" lingering.